nuxt/components/form/Form.vue

```diff
@@ -6,12 +6,21 @@
       formClass,
     ]"
     novalidate
-    @submit="(e) => emit('submit', e)"
+    @submit="(e) => onSubmit(e)"
```

Comment:
```suggestion
    @submit="onSubmit"
```
should be equivalent

Reply:
changed it in https://github.com/maevsi/maevsi/pull/1122/commits/494dc05e2eb41de0abcb2290071fd48aae9beee3